wai-aria/tree/basic.tentative.html

```diff
@@ -0,0 +1,39 @@
+<!doctype html>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+<script src="/resources/testdriver-actions.js"></script>
+<script src="/wai-aria/scripts/aria-utils.js"></script>
+
+<div id="listbox" role="listbox" aria-label="listbox">
```

Comment:
I think we should use a simpler test than listbox, primary because native listboxes (HTML select) are treated differently per platform.... (e.g. on Mac these are popup buttons that spawn menus)

How about a link with an image descendent, or a heading > link > image?

Reply:
Tablist seems okay for now. Thanks.